Index: transmission-1.80~b5/gtk/main.c
===================================================================
--- transmission-1.80~b5.orig/gtk/main.c	2010-01-21 17:58:16.000000000 +0000
+++ transmission-1.80~b5/gtk/main.c	2010-01-21 18:05:48.000000000 +0000
@@ -35,6 +35,10 @@
 #include <glib/gi18n.h>
 #include <glib/gstdio.h>
 
+#ifdef HAVE_LPI
+#include <launchpad-integration.h>
+#endif
+
 #include <libtransmission/transmission.h>
 #include <libtransmission/rpcimpl.h>
 #include <libtransmission/utils.h>
@@ -603,6 +607,10 @@
         myUIManager = gtk_ui_manager_new ( );
         actions_init ( myUIManager, cbdata );
         gtk_ui_manager_add_ui_from_string ( myUIManager, fallback_ui_file, -1, NULL );
+#ifdef HAVE_LPI
+        launchpad_integration_add_ui ( myUIManager,
+                                        "/main-window-menu/help-menu/launchpad-items" );
+#endif
         gtk_ui_manager_ensure_update ( myUIManager );
         gtk_window_set_default_icon_name ( MY_NAME );
 
Index: transmission-1.80~b5/gtk/ui.h
===================================================================
--- transmission-1.80~b5.orig/gtk/ui.h	2010-01-21 17:58:16.000000000 +0000
+++ transmission-1.80~b5/gtk/ui.h	2010-01-21 18:06:36.000000000 +0000
@@ -57,6 +57,9 @@
     "      <separator/>\n"
     "      <menuitem action='donate'/>\n"
     "      <separator/>\n"
+#ifdef HAVE_LPI
+    "      <placeholder name='launchpad-items' />\n"
+#endif
     "      <menuitem action='help'/>\n"
     "      <menuitem action='show-about-dialog'/>\n"
     "    </menu>\n"
Index: transmission-1.80~b5/gtk/Makefile.am
===================================================================
--- transmission-1.80~b5.orig/gtk/Makefile.am	2010-01-21 17:58:16.000000000 +0000
+++ transmission-1.80~b5/gtk/Makefile.am	2010-01-21 17:59:53.000000000 +0000
@@ -101,6 +101,7 @@
 dist_man_MANS = transmission.1
 
 transmission_LDADD = \
+    @LPI_LIBS@ \
     $(top_builddir)/libtransmission/libtransmission.a \
     $(top_builddir)/third-party/miniupnp/libminiupnp.a \
     $(top_builddir)/third-party/libnatpmp/libnatpmp.a \
Index: transmission-1.80~b5/configure.ac
===================================================================
--- transmission-1.80~b5.orig/configure.ac	2010-01-21 18:00:06.000000000 +0000
+++ transmission-1.80~b5/configure.ac	2010-01-21 18:03:53.000000000 +0000
@@ -248,6 +248,14 @@
 
 if test "x$build_gtk" = "xyes"; then
 
+    PKG_CHECK_MODULES([LPI],
+                      [launchpad-integration],
+                      [use_lpi=yes],
+                      [use_lpi=no])
+    if test "x$use_lpi" = "xyes"; then
+        AC_DEFINE([HAVE_LPI], 1)
+    fi
+	
     PKG_CHECK_MODULES([GIO],
                       [gio-2.0 >= $GIO_MINIMUM],
                       [use_gio=yes],
