Index: transmission-1.80~b5/third-party/Makefile.am
===================================================================
--- transmission-1.80~b5.orig/third-party/Makefile.am	2010-01-13 01:00:30.000000000 +0000
+++ transmission-1.80~b5/third-party/Makefile.am	2010-01-21 18:19:32.000000000 +0000
@@ -1,4 +1,4 @@
-SUBDIRS = libnatpmp miniupnp libevent dht
+SUBDIRS = libnatpmp miniupnp dht
 
 EXTRA_DIST = \
   macosx-libevent-config.h \
Index: transmission-1.80~b5/third-party/Makefile.in
===================================================================
--- transmission-1.80~b5.orig/third-party/Makefile.in	2010-01-13 01:34:23.000000000 +0000
+++ transmission-1.80~b5/third-party/Makefile.in	2010-01-21 18:19:32.000000000 +0000
@@ -279,7 +279,7 @@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
 transmissionlocaledir = @transmissionlocaledir@
-SUBDIRS = libnatpmp miniupnp libevent dht
+SUBDIRS = libnatpmp miniupnp dht
 EXTRA_DIST = \
   macosx-libevent-config.h \
   macosx-libevent-event-config.h
Index: transmission-1.80~b5/configure.ac
===================================================================
--- transmission-1.80~b5.orig/configure.ac	2010-01-21 18:19:43.000000000 +0000
+++ transmission-1.80~b5/configure.ac	2010-01-21 18:20:56.000000000 +0000
@@ -179,7 +179,7 @@
 dnl build our copy of libevent whether we use it or not,
 dnl because "make dist" needs its Makefiles to exist, and
 dnl AM_CONDITIONAL + AC_CONFIG_SUBDIRS don't seem to play nice
-AC_CONFIG_SUBDIRS([third-party/libevent])
+#AC_CONFIG_SUBDIRS([third-party/libevent])
 libevent_source=bundled
 AC_CHECK_LIB([event],[evutil_vsnprintf],
              [libevent_found=yes],
