Index: transmission-1.74/qt/qtr.pro
===================================================================
--- transmission-1.74.orig/qt/qtr.pro	2009-08-25 00:42:49.000000000 +0100
+++ transmission-1.74/qt/qtr.pro	2009-08-27 19:38:06.000000000 +0100
@@ -17,7 +17,7 @@
 LIBS += $${TRANSMISSION_TOP}/third-party/dht/libdht.a
 LIBS += $${TRANSMISSION_TOP}/third-party/miniupnp/libminiupnp.a
 LIBS += $${TRANSMISSION_TOP}/third-party/libnatpmp/libnatpmp.a
-LIBS += $${TRANSMISSION_TOP}/third-party/libevent/.libs/libevent.a
+LIBS += -levent
 
 TRANSLATIONS += transmission_en.ts transmission_ru.ts
 
